project/First-Project/src/main/resources/static/javascript/questionJS.js

```diff
@@ -19,11 +29,22 @@ $(document).ready(function () {
             data: JSON.stringify(question),
             contentType: "application/JSON",
             success: function (data) {
+                function sleep(milliseconds) {
+                    return new Promise(resolve => setTimeout(resolve, milliseconds));
```

Comment:
Why timeout and sleep before redirect?

Reply:
To get success message after posting a question for 3 seconds, then reload the page.